buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -0,0 +1,140 @@
+from pathlib import Path
+
+import numpy as np
+from pynwb.file import NWBFile, TimeIntervals, TimeSeries
+from pynwb.behavior import SpatialSeries, Position, CompassDirection
+from hdmf.backends.hdf5.h5_utils import H5DataIO
+
+from neuroconv.utils.json_schema import FolderPathType
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.tools.nwb_helpers import get_module
+
+from scipy.io import loadmat as loadmat_scipy
+
+
+class HuzsarBehaviorInterface(BaseDataInterface):
+    """Behavior interface"""
+
+    def __init__(self, folder_path: FolderPathType):
+        super().__init__(folder_path=folder_path)
+
+    def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = False):
+        self.session_path = Path(self.source_data["folder_path"])
+        self.session_id = self.session_path.stem
+
+        self.add_8_maze_behavior_data(nwbfile)
+        self.add_sleep_states(nwbfile)
```

Comment:
It also makes the conversion itself flatter, rather than one interface creating a multitude of neurodata objects from multiple streams, each interface is responsible for handling one type of data stream

Reply:
Yeah, I think that would make sense. In fact, I did not jump immediatley into that because I could not think a good name for the interfaces (any suggestion). Just too used to have everything sort of thrown together as Behavior. Do let me know if you any suggestions.

Reply:
Probably `HuszarSleepInterface` and `HuszarPositionInterface` since these might be somewhat specific to this experiment.

Though I had thought the sleep state one was general enough (since was identical to previous conversions?) to be considered a non-Huszar specific one. I might be wrong about that though

Reply:
Ok, this is done.